include/setup.h

```diff
@@ -150,17 +150,23 @@ class Property {
 
 	void Set_values(const char * const * in);
 	void Set_values(const std::vector<std::string> &in);
-	void Set_help(std::string const& str);
+	void Set_help(const std::string &str);
 
 	const char* GetHelp() const;
 
-	virtual	bool SetValue(std::string const& str)=0;
-	Value const& GetValue() const { return value;}
-	Value const& Get_Default_Value() const { return default_value; }
-	//CheckValue returns true, if value is in suggested_values;
-	//Type specific properties are encouraged to override this and check for type
-	//specific features.
-	virtual bool CheckValue(Value const& in, bool warn);
+	virtual bool SetValue(const std::string &str) = 0;
+	const Value &GetValue() const
+	{
+		return value;
+	}
+	const Value &Get_Default_Value() const
```

Comment:
While you're at it, change to GetDefaultValue?

Reply:
Got it; thanks!